bosh-setup/azuredeploy.json

```diff
@@ -66,6 +66,7 @@
   },
   "variables": {
     "apiVersion": "2015-06-15",
```

Comment:
I am following https://github.com/Azure/azure-quickstart-templates/issues/934, and it works well. The template can pass the template validation, and can be deployed successfully. So, I prefer to keep apiVersion as a var.